pairing/bls12381/kilic/group.go

```diff
@@ -0,0 +1,89 @@
+package kilic
+
+import (
+	"crypto/cipher"
+	"crypto/sha256"
+	"hash"
+
+	"go.dedis.ch/kyber/v3"
+	"go.dedis.ch/kyber/v3/util/random"
+	"go.dedis.ch/kyber/v3/xof/blake2xb"
+)
+
+// GroupChecker allows to verify if a Point is in the correct group or not. For
+// curves which don't have a prime order, we need to only consider the points
+// lying in the subgroup of prime order. That check returns true if the point is
+// correct or not.
+type GroupChecker interface {
+	kyber.Point
+	IsInCorrectGroup() bool
+}
```

Comment:
Was that 👍🏻  a "yes we need to export it" or a "I'll make it a non-exported or a test-only interface"? xD

Reply:
Yeah sorry forgot to remove it..it was used because kilic wasn't using the SubGroupElement used in kyber, so I just used that instead, now the GroupChecker can simply be removed instead of unexporting it :)